sentinel-demo/sentinel-demo-parameter-flow-control/src/main/java/com/alibaba/csp/sentinel/demo/flow/relate/RelationFlowControlDemo.java

```diff
@@ -0,0 +1,50 @@
+package com.alibaba.csp.sentinel.demo.flow.relate;
```

Comment:
Please add license header here (and in every new class).

Reply:
ok, I had done it.